copycat/api/src/main/java/org/apache/kafka/copycat/sink/SinkTask.java

```diff
@@ -62,4 +62,11 @@ public void initialize(SinkTaskContext context) {
      * @param offsets mapping of TopicPartition to committed offset
      */
     public abstract void flush(Map<TopicPartition, OffsetAndMetadata> offsets);
+
+
+    public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
```

Comment:
undocumented public API? Connector developers will need to know what to implement here.